### DISCUSSION TITLE
3D CAD models?

### DISCUSSION THREAD

**Discussion by laser-dance** [bitcraze/discussions#356] [Q&A]
Hello,
I'm wondering if there are 3D CAD models of the boards (Bolt, Flow, Micro SD) available. I couldn't find anything on them, if someone has found them and can point me towards that would be much appreciated.

**Comment by krichardsson** [comment]
There are some files in https://github.com/bitcraze/bitcraze-mechanics but not a lot I'm afraid.

@knmcguire Do you have anything from the simulator?

**Reply by knmcguire** [reply]
sorry for the delay. Yes you can uses these dae files here: https://github.com/bitcraze/crazyflie-simulation
